### ISSUE TITLE
Threat on the ICC boat event

### ISSUE THREAD

**Issue by Saiska** [trickerer/Trinity-Bots#121]
Like this one: https://github.com/trickerer/Trinity-Bots/issues/106

Bots don't generate aggro on Saurfang (after a first wipe, maybe it played on the bug, i forgot the rocket on the first try so i didn't see if aggro was generated)

edit correction: i said saurfang, in fact its the all the enemies.

**Comment by Saiska** [comment]
another weird thing with the dps sign, on sindragosa this time, the bots won't focus the ice tombe is sindragosa is far or flying. Like if i put the dps sign some milliseconds too late and the bots are not attacking something, they won't resume attacking on the sign. If they are currently attacking sindra and you ask them to switch they do, np.

**Comment by trickerer** [comment]
In all my ICC runs I couldn't reproduce the threat bug. Looks like a targeting icons conflict.
After ICC cannons fix I was able to do ICC Gunship Battle legit (2 tries).
After Sindragosa Ice Tomb targeting fix I did her without much trouble (as a healer, 'Avoid frontal AoE' positioning is required).

https://github.com/trickerer/Trinity-Bots/commit/faf36803e13e8c298b7ab92ad15cb8b7e0ea792b
https://github.com/trickerer/Trinity-Bots/commit/d00fb79a0211b48757b1a5f2c809f1d05e605c6e
https://github.com/trickerer/Trinity-Bots/commit/6d69e274d76dc93ab9a0932d1b27c1b11d72c2b9

Closing as solved
